src/EnergyPlus/SystemReports.cc

```diff
@@ -2284,7 +2284,18 @@ namespace SystemReports {
                                     PrimaryAirSystem(SysIndex).Name);
             }
         }
-
+        SetupOutputVariable("Air System Relief Air Total Heat Loss Energy",
+                            OutputProcessor::Unit::GJ,
```

Comment:
Just realized that a new unit has been introduced for output variables. J is the standard unit everywhere else, not sure we want to add GJ here. @Myoldmopar ???

Reply:
I don't have any reason why we need to add another unit here.  This should be joules.